Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM austinpray/python-node:3.4-8.12.0
+
+WORKDIR /soundslike
+
+COPY ./ /soundslike
+
+RUN pip install -r requirements.txt
+RUN npm install
```

Comment:
you prolly want `npm ci`

Reply:
Lol this app was created before `package-lock.json` was a thing 🙃 `npm ci` fails without one. Needta freeze my deps I guess